code/modules/lighting/_lighting_defs.dm

```diff
@@ -1,33 +1,43 @@
 // This is the define used to calculate falloff.
-#define LUM_FALLOFF(C, T) (1 - CLAMP01(sqrt((C.x - T.x) ** 2 + (C.y - T.y) ** 2 + LIGHTING_HEIGHT) / max(1, actual_range)))
-#define LUM_FALLOFF_XY(Cx,Cy,Tx,Ty) (1 - CLAMP01(sqrt(((Cx) - (Tx)) ** 2 + ((Cy) - (Ty)) ** 2 + LIGHTING_HEIGHT) / max(1, actual_range)))
+#define LUM_FALLOFF(Cx,Cy,Tx,Ty,HEIGHT) (1 - CLAMP01(sqrt(((Cx) - (Tx)) ** 2 + ((Cy) - (Ty)) ** 2 + HEIGHT) / max(1, actual_range)))
+
+// Simplified version of LUM_FALLOFF that only applies height falloff.
+#define HEIGHT_FALLOFF(HEIGHT) (1 - CLAMP01(sqrt(HEIGHT) / max(1, light_range)))
```

Comment:
This doesn't seem to be causing a measurable amount of falloff like `LUM_FALLOFF` does, but I don't want to use `LUM_FALLOFF` for sunlight sources as they're supposed to be uniform across the same Z.